WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -1042,13 +1042,6 @@ private extension ProductFormViewController {
             return
         }
 
-        let stores = ServiceLocator.stores
-        guard let site = stores.sessionManager.defaultSite,
-            stores.shouldAuthenticateAdminPage(for: site) else {
-            WebviewHelper.launch(url.absoluteString, with: self)
-            return
-        }
```

Comment:
Using authenticated web view controller now for all product preview - the web view now can check to see if authentication is possible. If not, it would just open the original URL.